src/app/(proper_react)/redesign/(public)/ValuePropositions.tsx

```diff
@@ -0,0 +1,514 @@
+          fill="url(#paint3_linear_2_211)"
+        />
+        <path
+          d="M199.627 108.871V116.018H203.66V108.539C203.658 106.82 202.735 105.234 201.242 104.382L197.237 102.088L191.65 98.8906C190.366 98.1571 188.796 98.1335 187.49 98.828L184.695 100.313C185.123 100.071 185.647 100.071 186.075 100.313L198.937 107.679C199.363 107.924 199.626 108.379 199.627 108.871Z"
+          fill="url(#paint4_linear_2_211)"
+        />
+        <path
+          opacity="0.9"
+          d="M199.626 117.169V123.599C199.625 124.091 199.361 124.545 198.933 124.788L194.583 127.28L195.261 128.365C196.073 129.66 197.763 130.081 199.088 129.319L201.242 128.087C202.737 127.234 203.659 125.645 203.66 123.923V117.169H199.626Z"
+          fill="url(#paint5_linear_2_211)"
+        />
+        <path
+          opacity="0.9"
+          d="M194.274 126.813L192.387 123.969C191.376 124.871 190.2 125.57 188.923 126.025L190.93 129.053C192.063 128.331 193.178 127.584 194.274 126.813Z"
+          fill="url(#paint6_linear_2_211)"
+        />
+        <text
+          fill="black"
+          style={{ whiteSpace: "pre" }}
+          fontFamily="var(--font-metropolis)"
+          fontSize="29.2252"
+          fontWeight="bold"
+          letterSpacing="-0.208751px"
+        >
+          <tspan x="218.275" y="126.429">
+            Mozilla{" "}
+          </tspan>
+        </text>
+        <text
+          fill="black"
+          style={{ whiteSpace: "pre" }}
+          fontFamily="var(--font-metropolis)"
+          fontSize="29.2252"
+          fontWeight="500"
+          letterSpacing="-0.626254px"
+        >
+          <tspan x="325.115" y="126.429">
+            Monitor
+          </tspan>
+        </text>
+        <path
+          d="M96.406 175.79C96.406 172.67 98.9355 170.141 102.056 170.141H497.795C500.916 170.141 503.445 172.67 503.445 175.79V661.862C503.445 664.982 500.916 667.512 497.795 667.512H102.056C98.9354 667.512 96.406 664.982 96.406 661.862V175.79Z"
+          fill="white"
+        />
+        <path
+          d="M497.795 169.788H102.056C98.7404 169.788 96.0529 172.475 96.0529 175.79V661.862C96.0529 665.177 98.7404 667.865 102.056 667.865H497.795C501.111 667.865 503.798 665.177 503.798 661.862V175.79C503.798 172.475 501.111 169.788 497.795 169.788Z"
+          stroke="black"
+          stroke-opacity="0.1"
+          stroke-width="0.706212"
+        />
+        <text
+          fill="black"
+          style={{ whiteSpace: "pre" }}
+          fontFamily="var(--font-metropolis)"
+          fontSize="28"
+          fontWeight="bold"
+          letterSpacing="0px"
+        >
+          <tspan x="124" y="319.26">
+            Your glassesdotcom.com{" "}
+          </tspan>
+          <tspan x="124" y="353.26">
+            password was exposed
+          </tspan>
```

Comment:
I don't have full context yet, so one thing you'll need to consider is if this will still work if the localised strings are much shorter (e.g. Japanese, I suppose) or longer (e.g. German or Dutch), but you could potentially split it up like this?

```suggestion
          {l10n.getFragment("string-id", {
              elems: {
                "first-line": <tspan x="124" y="319.26"/>,
                "second-line": <tspan x="124" y="353.26"/>
              },
          })};
```

And then the Fluent string would look like this:

```ftl
string-id = <first-line>Your glassesdotcom.com</first-line> <second-line>password was exposed</second-line>
```

Though also cc @flodolo in case he has other ideas.